operations/observability/mixins/jsonnetfile.json

```
@@ -19,6 +19,15 @@
       },
       "version": "master",
       "name": "unit"
+    },
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/povilasv/coredns-mixin.git",
+          "subdir": "dashboards"
```

```suggestion
          "subdir": ""
```

The subdir must be the directory where `jsonnetfile.json` is located in that repo. For coredns-mixin it is the root dir :)